workflows/rnafusion.nf

```diff
@@ -45,136 +48,215 @@ workflow RNAFUSION {
     // Create references if necessary
     //
 
-    BUILD_REFERENCES()
+    BUILD_REFERENCES(tools)
     ch_versions = ch_versions.mix(BUILD_REFERENCES.out.versions)
 
     if (!params.references_only) { // TODO: Remove this temporary parameter when we have a full-working GitHub nf-test
```

Comment:
```suggestion
    if (!params.references_only) {
```

Reply:
this is not a temporary parameter but one that allows user to only build the references. As it takes ages and for documentation purposes, this behavior is wished. 

(not related to this PR but since I saw the comment, why not correct it 😄  )

Reply:
Fair enough, thanks ;)